import-job.sh

```diff
@@ -1,5 +1,12 @@
 #!/bin/sh
+
+if [ ! -z $1 ]; then
+   JOB=$1
+else
+   JOB="IQSS-dataverse-develop"
```

Comment:
```suggestion
   JOB="IQSS-Dataverse-Develop"
```

To avoid this: ./import-job.sh: line 12: IQSS-dataverse-develop.xml: No such file or directory